301-web-app-regional-vnet-private-endpoint-sql-storage/nestedtemplates/private_link_ipconfigs.json

```diff
@@ -0,0 +1,54 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+      "privateDnsZoneName": {
+        "type":"string",
+        "metadata":{
+          "description": "private dns zone name"
+        }
+      },
+      "privateLinkNicResource": {
+        "type":"string",
+        "metadata":{
+          "description": "private endpoint Nic ipConfigurations array resource id"
+        }
+      },
+      "privateLinkNicIpConfigTemplateUri": {
+        "type":"string",
+        "metadata":{
+          "description": "ip configurations template resource URI"
+        }
+      },
+      "privateDnsRecordTemplateUri": {
+        "type":"string",
+        "metadata":{
+          "description": "private dns record template resource URI"
+        }
+      }
+    },
+    "variables": {
+    },
+    "resources": [
+      {
+        "type": "Microsoft.Resources/deployments",
+        "apiVersion": "2019-10-01",
+        "name": "private-link-ipconfigs",
+        "properties": {

```

Comment:
move properties to the bottom of the resource definition

Reply:
Ok, I have made the requested change.